tests/test_glue/test_glue.py

```diff
@@ -41,6 +42,26 @@ def test_list_jobs():
     response.shouldnt.have.key("NextToken")
 
 
+@mock_glue
+def test_get_job_not_exists():
+    client = create_glue_client()
+    name = "my_job_name"
+
+    with pytest.raises(ClientError) as exc:
+        client.get_job(JobName=name)
+
+    exc.value.response["Error"]["Code"].should.equal("EntityNotFoundException")
+    exc.value.response["Error"]["Message"].should.match("Job my_job_name not found")
+
+
+@mock_glue
+def test_get_job_exists():
+    client = create_glue_client()
+    job_name = create_test_job(client)
+    response = client.get_job(JobName=job_name)
+    assert response["Job"]["Name"] == job_name
```

Comment:
Can we get a few more assertions here, just to avoid the chance of any other typo's?

Reply:
yes on the way